src/sentry/integrations/slack/message_builder/base/block.py

```diff
@@ -37,19 +37,20 @@ def get_divider() -> SlackBlock:
         return {"type": "divider"}
 
     @staticmethod
-    def get_action_block(actions: Sequence[Tuple[str, str, str]]) -> SlackBlock:
-        return {
-            "type": "actions",
-            "elements": [
-                {
-                    "type": "button",
-                    "text": {"type": "plain_text", "text": text},
-                    "url": url,
-                    "value": value,
-                }
-                for text, url, value in actions
-            ],
-        }
+    def get_action_block(actions: Sequence[Tuple[str, Optional[str], str]]) -> SlackBlock:
+        action_block = {"type": "actions", "elements": []}
+        for text, url, value in actions:
+            button = {
+                "type": "button",
+                "text": {"type": "plain_text", "text": text},
+                "value": value,
+            }
+            if url:
+                button["url"] = url
```

Comment:
Made the url optional because if we pass None here, it results in an invalid_payload error